CONTRIBUTING.md

```diff
@@ -76,26 +58,117 @@ the WSL2 subsystem with the dev container.
     1. Install .NET Core: `sudo apt-get update && sudo apt-get install -y dotnet-runtime-7.0`
     1. Restart the WSL terminal.
     1. Create a directory for source code and `cd` into it.
-    1. Clone or fork the repo as described below in
-       [Contributing to Code and Documentation](#contributing-to-code-and-documentation)
+    1. Clone or fork the repo as described in [Contributing to Code and Documentation](#contributing-to-code-and-documentation)
     1. `cd` into the repo's directory
-    1. Open VSCode from the command line with: `code .`
-    1. From the command window (Ctrl + Shift + P) choose `Reopen in Container`
+    1. Open VSCode from the command line with: `code .` (or `code-insiders .` if using the Insiders build)
```

Comment:
Please link to a vs code insiders page that explains what the Insiders build is.

Reply:
done! 👍